### PR TITLE
Show 0 instead of null for lowest bucket

### PR DESCRIPTION
**Pull request by lilyfromseattle**
__Before:__
Screen Shot 2019-04-10 at 1 19 48 PM

__After:__
Screen Shot 2019-04-10 at 1 23 07 PM

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit b4043d4295f9a2a7784340d68f27d0710cf45f18

https://deploy-preview-2287--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/PricingContext.tsx [hunk 1]
```diff
@@ -32,14 +32,19 @@ function PricingContext({ artwork }: PricingContextProps) {
       </Flex>
       <Spacer mb={[2, 3]} />
       <BarChart
-        minLabel={artwork.pricingContext.bins[0].minPrice}
+        minLabel={
+          artwork.pricingContext.bins[0].minPrice != null
```
Comment (sweir27) [inline src/Apps/Artwork/Components/PricingContext.tsx]: It looks like from vortex, we're returning a `0` correctly:
image

From metaphysics, we're returning a `null`:
image

So it seems in metaphysics we should be converting that `0` cents value into `$0` but aren't doing so. I remembered encountering this issue before: https://github.com/artsy/metaphysics/pull/1325 but I didn't end up revisiting like I said I would, and now it's biting us again!

Could you add a `TODO` here so we can revisit? I'll add an item to our sprint to revisit the metaphysics work, but that can be a separate work-item.

Comment (lilyfromseattle) [reply src/Apps/Artwork/Components/PricingContext.tsx]: Will do

**Comment by artsyit** [comment]
:rocket: PR was released in v15.14.4 :rocket:

**Pull request merged**